src/utils/rest-api.js

```diff
@@ -710,6 +710,25 @@ export function updateTreeNode(node) {
     );
 }
 
+export function deleteModification(studyUuid, node, modificationUuid) {
+    const modificationDeleteUrl =
+        PREFIX_STUDY_QUERIES +
+        '/v1/studies/' +
+        studyUuid +
+        '/nodes/' +
+        encodeURIComponent(node.id) +
+        '/network-modification/' +
```

Comment:
/network-modification**s**/ (sync with back modification)

Reply:
there is already a put without s
    @PutMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/loads")
I use the same path but with a delete

Reply:
for a change like this it will need a separate PR

Reply:
I'm telling this because of API rules we defined you know. Here for the new API and a fix for the other one or in another PR for both of them it's ok for me.

Reply:
I think it would be better to make it right in this PR

Reply:
@niconoir , if we do this and then merge the feature, it can't work until we update the API from back side.
We have a ticket to apply our API REST best practices in everyf services, it will be done in this context.